Neos.Flow/Classes/Persistence/Doctrine/PersistenceManager.php

```diff
@@ -98,51 +90,11 @@ public function persistAll($onlyAllowedObjects = false)
             return;
         }
 
-        $this->onlyAllowedObjects = $onlyAllowedObjects;
+        $this->allowedObjects->checkNext($onlyAllowedObjects);
         $this->entityManager->flush();
-        $this->onlyAllowedObjects = false;
```

Comment:
This false assignment here was weird anyway. Having that "one-time" logic encapsulated in the `AllowedObjectsContainer` is an improvement.